neurom/io/check.py

```diff
@@ -79,7 +77,7 @@ def has_increasing_ids(raw_data):
     with their predecessor)
     '''
     ids = raw_data.get_col(COLS.ID)
-    steps = [int(j) for (i, j) in zip(ids, ids[1:]) if i >= j]
+    steps = ids[np.where(np.diff(ids) <= 0)[0] + 1].astype(int)
```

Comment:
Is the dtype conversion in the end necessary? If the steps are used by numpy arrays, I think it does not matter if the are floats as long as they do not have decimals.